tests/foreman/api/test_errata.py

```diff
@@ -158,11 +159,11 @@ def test_positive_install_in_hc(module_org, activation_key, custom_repo, target_
 
 
 @pytest.mark.tier3
-@pytest.mark.rhel_ver_list([7, 8, 9])
+@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
```

Comment:
Can you translate this for me?

Reply:
@Griffin-Sullivan this was recommended in other parameterized tests I've written.
It will automatically select any version higher than rhel6, so it should find 10 once it is out!
I admit it's a bit difficult on the eyes xD

Reply:
@sambible oops, you are right thanks!

Reply:
I'm not 100% sure, but I think you could also do it like this but just replace the 6 with a 7. https://github.com/SatelliteQE/robottelo/blob/0ddb4b712f659b78791e5a6437dca4bf140e8ce6/tests/foreman/ui/test_repositories.py#L22

Either way, I see other instances like this, so no harm in doing it this way.

Reply:
@Griffin-Sullivan I like your way a lot better! I'll push to this pr #12851